src/Sulu/Bundle/AdminBundle/Resources/js/views/FormOverlayList/README.md

```diff
@@ -0,0 +1,3 @@
+The form-overlay-list is registered with the key `sulu_admin.form_overlay_list`. 
+It shows a List container with a toolbar. The actions that are displayed in the toolbar can be configured using the route's options.
```

Comment:
We usually also list the available options in a table at the end of the README. It's hard to see them somewhere otherwise.

Reply:
i noticed that the available options in the other `README` files are kind of outdated. for example, the `README` of the `List` view contains only two options. therefore assumed that we dont update that anymore 🙈 . do we?

Reply:
Yeah, just because we might have forgotten that in other places, I would say that's not an excuse to stop doing it at all.

Reply:
added the documentation for the `list` view and the `form-overlay-list` view